monochart/templates/job.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
 {{- end }}
 spec:
   activeDeadlineSeconds: {{ default 300 $job.activeDeadlineSeconds }}
+  backoffLimit: {{ default 6 $job.backoffLimit }}
```

Comment:
Can this parameter be documented anywhere?  Either in the spoton-app readme or shown in the values or example values file would probably be appropriate.